tensorflow/lite/CMakeLists.txt

```diff
@@ -619,6 +618,7 @@ target_link_libraries(tensorflow-lite
     farmhash
     fft2d_fftsg2d
     flatbuffers::flatbuffers
+    gemmlowp::gemmlowp
```

Comment:
This seems to undo github.com/tensorflow/tensorflow/pull/58678. Have you checked this isn't a problem?

Reply:
My understanding is, that #58678 was created because linking against `gemmlowp` only works when `gemmlowp` is downloaded by tensorflow. When building with system libraries and headers `find_package(gemmlowp)` works but the target `gemmlowp` doesn't exist as it prefixed (`gemmlowp::gemmlowp`). Therefore `-lgemmlowp` is added as CMake has no other information. I guess the idea was one would supply the include path for `gemmlowp`. But the original proposal in #58678 broke internal builds therefore the `target_include_directories()` change was added, which wasn't part of the original PR.

The proposed and now merged way ensures that `gemmlowp::gemmlowp` exists no matter the origin. And because of the prefix it should always be a CMake target/library, in this case a interface library.

I tested this PR both ways, by downloading gemmlowp externally and through the config step of tensorflow and both are able to find and include gemmlowp.